components/layout/user-nav.tsx

```diff
@@ -53,15 +54,13 @@ export function UserNav() {
             Profile
             <DropdownMenuShortcut>⇧⌘P</DropdownMenuShortcut>
           </DropdownMenuItem>
-          <DropdownMenuItem>
+          {session?.user.accountType === ACCOUNT_TYPES.USER ? <DropdownMenuItem>
```

Comment:
use the "&&" check here since your condition check is not returning any values if the condition is not met. this way it makes ot easy to undderstand the code